src/main/java/ch/kalunight/zoe/command/remove/RemoveAccountCommand.java

```diff
@@ -34,37 +38,40 @@ public RemoveAccountCommand() {
   }
   
   @Override
-  protected void executeCommand(CommandEvent event) {
+  protected void executeCommand(CommandEvent event) throws SQLException {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'executeCommand(CommandEvent)' has an NPath complexity of 768](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)